apis/dialapi/dialapi.yaml

```diff
@@ -1202,14 +1451,14 @@ paths:
             to contact us. In the meantime, try refreshing.
           content: {}
       deprecated: false
-      security: 
+      security:
         - bearer: []
   /dialcode/v1/reserve:
     post:
       tags:
       - DIAL APIs
       summary: Reserve DIAL code
-      description: 
+      description:
```

Comment:
1. Please check the formatting of the description and the bullet points. They are appearing in a single paragraph instead of bullet points.

2. Please change "It" to "It's" in line 1470.

Reply:
Dont know what to do for formatting. I have added space to see if it resolves the problem. Please verify